src/ir/value.cpp

```diff
@@ -103,30 +103,13 @@ Const* Const_impl<Module*>(Context* c,Module* val) {
   return c->valuecache->getModule(val);
 }
 
-//template<>
-//string TemplatedConst<bool>::toString() const {
-//  return vtype->toString() + "(" + (value ? "True" : "False") + ")";
-//}
-//
-//template<>
-//string TemplatedConst<int>::toString() const {
-//  return vtype->toString() + "(" + to_string(value) + ")";
-//}
-//
-//template<>
-//string TemplatedConst<BitVector>::toString() const {
-//  return vtype->toString() + "(" + to_string(value.to_type<int>()) + ")";
-//}
-//
-//template<>
-//string TemplatedConst<std::string>::toString() const {
-//  return vtype->toString() + "(" + value + ")";
-//}
-//
-//template<>
-//string TemplatedConst<Type*>::toString() const {
-//  return vtype->toString() + "(" + value->toString() + ")";
-//}
```

Comment:
All these toString methods are being removed?

Reply:
Similar to the other comment, these were being unused. there was already a toString method defined for this class.